### PR TITLE
[#2197] Remove carousel from adopters list.

### PR DESCRIPTION
**Pull request by b-abel**
This removes the carousel from the adopter's section on the homepage to increase the stability. The logos are now simply displayed as a vertical list with automatically adjusts to small window sizes.

### CODE REVIEW

**Comment by sophokles73** [comment]
Is it possible to have the whole block centered horizontally and have each logo centered vertically? Currently, the RedHat logo seems to be aligned to the top ...

**Comment by b-abel** [comment]
The new solution should be released [next month](https://github.com/EclipseFdn/iot.eclipse.org/issues/526#issuecomment-696931507) by the Eclipse Foundation. Since I have no idea how it differs from the current one, I wonder if we shouldn't wait on that before we change something on our side. Maybe we should put this PR on hold for now?

**Comment by sophokles73** [comment]
> Maybe we should put this PR on hold for now?

I agree. It's (hopefully) just a few days more until the fix will be available ...

**Comment by sophokles73** [comment]
@b-abel Has the fix been deployed by the EF?

**Comment by b-abel** [comment]
@sophokles73 The script has been fixed for now. The new (and hopefully more stable) solution has not been published yet, as far as I know. I would still wait for it and then see what we have to do on our site to adapt to it.

**Comment by sophokles73** [comment]
@b-abel another month goes by ... what's the status of this PR?

**Comment by b-abel** [comment]
I still have not seen any release of a new solution by the Eclipse Foundation. I suggest that we merge this PR now.

**Comment by sophokles73** [comment]
what about 
> Is it possible to have the whole block centered horizontally and have each logo centered vertically? Currently, the RedHat logo seems to be aligned to the top ...

**Comment by b-abel** [comment]
> Is it possible to have the whole block centered horizontally and have each logo centered vertically? Currently, the RedHat logo seems to be aligned to the top ...

Please take a look at my last commit. I did not get the vertical centering of each logo with dynamic "line breaks" to work. My proposal now always shows the logos in two columns, regardless of screen size.

File: site/homepage/static/css/custom.css [hunk 1]
```diff
@@ -116,8 +116,16 @@ pre {
...
+}
+
+.eclipsefdn-adopters ul li {
+  display: inline-block;
+  padding: 2%;
+  width: 50%;
+}
+
+.eclipsefdn-adopters ul li a img {
+  width: 100%;
```
Comment (sophokles73) [inline site/homepage/static/css/custom.css:130]: I have tried this out locally and FMPOV it looks much better at 80%. WDYT?

Comment (b-abel) [reply site/homepage/static/css/custom.css]: Only reducing it to 80% leads to empty space on the right side. So, I centered the logos additionally in my last commit. WDYT?

**Review by sophokles73 — APPROVED** [review]
LGTM

**Pull request merged**